sdk/python/kfp/v2/compiler_cli_tests/test_data/two_step_pipeline_with_importer.py

```diff
@@ -28,7 +28,7 @@
 @dsl.pipeline(
     name='two-step-pipeline-with-importer',
     description='A linear two-step pipeline.')
-def my_pipeline(input_gcs='gs://test-bucket/pipeline_root',
+def my_pipeline(input_gcs = 'gs://test-bucket/pipeline_root',
```

Comment:
Will this untyped parameter be a problem?

Reply:
This was necessary for importer scenario. If we type annotated this as a `str`, and pass it to a component input typed as an artifact, compiler will complain about the type mis-matching (we can disable the type check, though probably not a good idea).

That said, this is indeed broken now as no artifact input allowed in the root component. I'll make a separate PR to attach importer using an explicit step.